### PR TITLE
Implement skipping filtering by list of attributes

### LINKED ISSUE #3513: FR: do not filter command results
Command result usually comes without real position, we construct it from last one. It is very likely it will be filtered by duplicate or distance.
What do you think about adding new switch to exclude from filtering positions with command results? Like we do for alarms #3484

### PR DESCRIPTION
**Pull request by Abyss777**
Main switch is `filter.skipAttributes.enable` because lookup might costs a lot.
Attributes list should be in `filter.skipAttributes`

fix #3513 

More comments in code

### CODE REVIEW

File: test/org/traccar/BaseTest.java [hunk 1]
```diff
@@ -40,25 +40,25 @@ public boolean isLatestPosition(Position position) {
...
                     long deviceId, String attributeName, boolean defaultValue, boolean lookupConfig) {
-                return false;
+                return defaultValue;
             }
 
             @Override
             public String lookupAttributeString(
                     long deviceId, String attributeName, String defaultValue, boolean lookupConfig) {
-                return null;
+                return defaultValue;
```
Comment (Abyss777) [inline test/org/traccar/BaseTest.java]: Some pieces of code might rely on `defaultValue` and not waited `null`

File: test/org/traccar/FilterHandlerTest.java [hunk 2]
```diff
@@ -3,14 +3,74 @@
...
 import java.util.Date;
 
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
-public class FilterHandlerTest extends BaseTest {
+public class FilterHandlerTest {
+
+    static {
+        Context.init(new IdentityManager() {
```
Comment (Abyss777) [inline test/org/traccar/FilterHandlerTest.java]: Have not found better way to override `lookupAttribute` returned value than totally reinitialize `IdentityManager`

File: src/org/traccar/FilterHandler.java [hunk 3]
```diff
@@ -145,8 +145,17 @@ private boolean skipLimit(Position position, Position last) {
         return false;
     }
 
-    private boolean skipAlarms(Position position) {
-        return skipAlarms && position.getAttributes().containsKey(Position.KEY_ALARM);
+    private boolean skipAttributes(Position position) {
+        if (skipAttributes) {
+            String attributesString = Context.getIdentityManager().lookupAttributeString(
+                    position.getDeviceId(), "filter.skipAttributes", "", true);
+            for (String attribute : attributesString.split("[ ,]")) {
```
Comment (tananaev) [inline src/org/traccar/FilterHandler.java]: I think we need to make it more efficient than parsing string every time. Maybe parse it into a list once when we create handler.

Comment (Abyss777) [reply src/org/traccar/FilterHandler.java]: We are looking up in devices tree, not only in config. It is impossible to "pre-parse"

Comment (tananaev) [reply src/org/traccar/FilterHandler.java]: Good point. I forgot about it.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**